src/Rokalo/Startup.cs

```diff
@@ -30,6 +32,8 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddCors();
             services.AddInfrastructureUsersConfiguration(MssqlSettings);
+            services.AddInfrastructureEmailConfiguration();
+            services.Configure<SmtpConfiguration>(Configuration.GetSection(SmtpConfiguration.Key));
```

Comment:
We should configure this in DependencyInjection of EmailConfiguration extension method